components/brave_wallet_ui/components/shared/add-custom-token-form/add-custom-token-form.tsx

```diff
@@ -223,6 +223,16 @@ export const AddCustomTokenForm = (props: Props) => {
     tokenDecimalsError
   ])
 
+  const tokenAlreadyExists = React.useMemo(() => {
+    if (tokenContractAddress !== '' && customAssetsNetwork !== undefined) {
+      return userVisibleTokensInfo.some((t) =>
+        t.contractAddress.toLocaleLowerCase() === tokenContractAddress.toLowerCase() &&
+        t.chainId === customAssetsNetwork?.chainId
+      )
+    }
+    return false
+  }, [tokenContractAddress, userVisibleTokensInfo, customAssetsNetwork])
```

Comment:
```suggestion
  const tokenAlreadyExists = React.useMemo(() => {
    if (tokenContractAddress !== '' && customAssetsNetwork?.chainId !== undefined) {
      return userVisibleTokensInfo.some((t) =>
        t.contractAddress.toLocaleLowerCase() === tokenContractAddress.toLowerCase() &&
        t.chainId === customAssetsNetwork?.chainId
      )
    }
    return false
  }, [tokenContractAddress, userVisibleTokensInfo, customAssetsNetwork?.chainId])
```

Might prevent some extra re-computes

Reply:
Updated!